src/debugpy/adapter/clients.py

```diff
@@ -770,7 +777,7 @@ def notify_of_subprocess(self, conn):
 
 def serve(host, port):
     global listener
-    listener = sockets.serve("Client", Client, host, port)
+    listener = sockets.serve("Client", Client, host, port) # type: ignore
```

Comment:
Maybe use `# pyright: ignore [specific diagnostic]`? Here and other places `type: ignore` is used.

Reply:
Wasn't there a way to do this automatically in Pylance? I thought we had a setting for it

Reply:
You need to have `enableTypeIgnoreComments` disabled.